bookkeeper-server/src/main/java/org/apache/bookkeeper/proto/PerChannelBookieClient.java

```diff
@@ -504,6 +504,12 @@ public boolean isSecure() {
         };
     }
 
+    public static String buildStatsLoggerScopeName(BookieId addr) {
+        StringBuilder nameBuilder = new StringBuilder();
+        nameBuilder.append(addr.toString().replace('.', '_').replace('-', '_').replace(":", "_"));
```

Comment:
Probably do not need StringBuilder for this.
Just need to return `addr.toString().replace('.', '_').replace('-', '_').replace(":", "_")`

Reply:
Yes. Addressed.